fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -586,6 +588,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_NEREIDS_STATS_DERIVE_V2)
     public boolean enableNereidsStatsDeriveV2 = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_ELIMINATE_SORT_NODE)
+    public boolean enableEliminateSortNode = true;
```

Comment:
Is it necessary to add this knob?

Reply:
The main concern is that there may be an unknown problem that can be recovered by turning it off